src/Microsoft.VisualStudio.AppDesigner/interop/NativeMethods.vb

```diff
@@ -5,6 +5,8 @@ Imports Microsoft.VisualStudio.OLE.Interop
 Imports ComTypes = System.Runtime.InteropServices.ComTypes
 Imports System.Windows.Forms
 
+#Disable Warning IDE1006 ' Naming Styles
```

Comment:
Worth dropping a .editorconfig file in this directory instead?

Reply:
Meh, probably not going to ever change this code again.